deploy/addon/pulsar.md

```diff
@@ -0,0 +1,268 @@
+---
+title: Pulsar
+shortdesc: A distributed messaging and streaming platform on the publish/subsribe model
+tags:
+- addons
+keywords:
+- pulsar
+- messaging
+- cold storage
+- queue
+- hot storage
+- biscuit
+- rabbitmq
+- kafka
+---
+
+## Overview
+
+Pulsar works on a publisher/subscriber model, just like RabbitMQ or Kafka.
+
```

Comment:
Before talking consuming strategies, we could talk on:

* why are we using pub/sub tech? 
* why do we love event-driven architecture?

Reply:
@PierreZ Can you contribute these points?

Reply:
I've done it on 542d67ed3e05f37baf5ca9b17339323f153998b4 and 1ae3ab7a3b8252957823202075500eed6067c30c.